src/modules/fancyzones/lib/FancyZonesData.cpp

```diff
@@ -463,7 +550,6 @@ void FancyZonesData::LoadFancyZonesData()
 {
     if (!std::filesystem::exists(zonesSettingsFileName))
     {
-        MigrateCustomZoneSetsFromRegistry();
```

Comment:
After removal of this migration, I think this if statement, actually body inside if is needed anymore.

Reply:
`SaveFancyZones` will just create files with empy json values. Without that, we would need to check for file.exist() when trying to pase